default/python/AI/buildings.py

```diff
@@ -64,3 +87,29 @@ def production_time(self, planet: PlanetId, empire: EmpireId = THIS_EMPIRE) -> i
         if empire is THIS_EMPIRE:
             empire = fo.empireID()
         return fo.getBuildingType(self.value).productionTime(empire, planet)
+
+    def turn_cost(self, planet: PlanetId, empire: EmpireId = THIS_EMPIRE) -> float:
+        return self.production_cost(planet, empire) / self.production_time(planet, empire)
```

Comment:
I wonder whether we really need the empire arguments. I doubt that the AI will ever use it.
I know, I should have asked this in the review of the last PR...